examples/supplychain-app/quorum/smartContracts/contracts/Permission.sol

```diff
@@ -1,4 +1,4 @@
-pragma solidity >0.6.0;
+pragma solidity 0.6.1;
```

Comment:
use the >0.6.0 otherwise there is compiler error when solidity version is anything else